### ISSUE TITLE
Question: Does additional_project_globs work with remote repository directories

### ISSUE THREAD

**Issue by victor-efedi-sympower** [spacelift-io/terraform-provider-spacelift#569]
Does `additional_project_globs` work with remote repository directories? 
Using terraform modules managed in another repository, would like to use `additional_project_globs` to trigger a stack if the remote terraform module changes.

**Comment by mbialon** [comment]
No, `additional_project_globs` apply only to the same repository.
